bode/bode/models/task.py

```diff
@@ -9,14 +11,23 @@
 from bode.models.utc_datetime import UTCDateTime
 
 
+class Status(enum.Enum):
+    todo = "todo"
+    indirectly_done = "indirectly_done"
+    done = "done"
```

Comment:
We use uppercase for enum members:
[Enum docs](https://docs.python.org/3/library/enum.html)

Reply:
And after you change that, you'll need to update some places to use `Status.STATUS.value` instead of `.name`